spec/services/search/jobseeker_profile_search_spec.rb

```diff
@@ -19,7 +19,7 @@
         let(:location_in_london) { [51.5072, -0.1275] }
         let(:london_jobseeker_profile) { create(:jobseeker_profile) }
         let(:london_job_preferences) { create(:job_preferences, jobseeker_profile: london_jobseeker_profile) }
-        let!(:london_location_preference) { create(:job_preferences_location, name: "London", radius: 100, job_preferences: london_job_preferences) }
+        let!(:london_location_preference) { create(:job_preferences_location, name: "London", radius: 150, job_preferences: london_job_preferences) }
```

Comment:
Increased the radius to ensure this location showed up after changing `Geocoder::DEFAULT_STUB_COORDINATES`